src/Build/Resources/Strings.resx

```diff
@@ -1208,21 +1208,9 @@
     <comment>{StrBegin="MSB4242: "}</comment>
   </data>
   <data name="CouldNotRunNuGetSdkResolver" UESanitized="false" Visibility="Public">
-    <value>MSB4243: The NuGet-based SDK resolver failed to run because NuGet assemblies could not be located.  Check your installation of MSBuild or set the environment variable "{0}" to the folder that contains the required NuGet assemblies. {1}</value>
+    <value>MSB4243: The NuGet-based SDK resolver failed to run because NuGet assemblies could not be located.  Check your installation of MSBuild or set the environment variable "MSBUILD_NUGET_PATH" to the folder that contains the required NuGet assemblies. {0}</value>
```

Comment:
Why not parameterized? Nice to have it in a single place.

Reply:
I've moved the environment variable name to MSBuildConstants and brought this back